### ISSUE TITLE
use iframes from core add-on options page for study consent forms

### ISSUE THREAD

**Issue by rhelmer** [mozilla-rally/rally-core-addon#399]
Per discussion with @hamilton and the add-ons team [in Slack](https://mozilla.slack.com/archives/C4C0HCX6F/p1613501037063100), as long as we're opening our options page in a new tab (`moz-extension://` URL), which we are, it *should* be feasible to use iframes to `rally.mozilla.org` to host study consent forms vs. embedding into the core add-on, or having to be restricted to a template.

**Comment by hamilton** [comment]
Note to self later: this should include a separate issue in the site repository to match the website work

**Comment by Dexterp37** [comment]
Note: @wagnerand needs to be consulted before moving on with this

**Comment by rhelmer** [comment]
> Note: @wagnerand needs to be consulted before moving on with this

Yes, this is something (per thread) I asked about feasibility of, but not something we should try to ship in the very near term. Worth a proposal IMHO.
